waffle-compiler/src/compiler.ts

```diff
@@ -30,7 +30,7 @@ async function newCompile(config: Config) {
     resolvers.BacktrackFsResolver(config.nodeModulesDirectory)
   );
   const sources = await gatherSources(
-    findInputs(config.sourceDirectory),
+    findInputs(config.sourceDirectory, ['.sol', '.vy']),
```

Comment:
#compilationtype
Should be `.vy` if configuration type is `dockerized-vyper`, `.sol` otherwise.
Extract function, e.g. `getExtenstionForCompilerType()`